tools/serverpod_cli/bin/util/print.dart

```diff
@@ -19,7 +20,23 @@ void printww(String string) {
   stdout.writeln(string);
 }
 
+/// Print to terminal with word wrap.
+void printww(String string) => _printww(string);
+
+/// Print to terminal with a new line.
 void printwwln(String string) {
   printww(string);
   printww('');
 }
+
+/// Print to terminal as warning in yellow color.
```

Comment:
I will not accept this change unless it is used consistently throughout the CLI tooling. It should also be a separate PR.

Reply:
Yes will make another PR for this part